WeatherService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Weather_Monitoring_System.Bots;
```

Comment:
Please put all services in a special folder for services called Services for example.

Reply:
Yes, I have this folder, I renamed the Bots folder to BotsServices, but I don't know why the visual doesn't reflect it, I will correct this issue, Or do you mean that I must put weatherService class also in this folder and rename the folder to Services?

Reply:
Yes, please put all services under the services folder. 
You can divide that folder for further subfolders as you want. But all services should be in the same layer (folder)